stylus/include/OurOfferings.styl

```diff
@@ -1,25 +1,33 @@
 .OurOfferings
+	margin-bottom: 5rem
 	margin-top: 5rem
 	text-align: center
 	>h1
 		font-family: Arvo
 		margin-bottom: 2rem
 	>p
+		margin-bottom: 5rem
 		margin-left: 33%
 		margin-right: 33%
 	>div
-		margin-bottom: 5rem
-		margin-left: 20rem
-		margin-right: 20rem
-		margin-top: 5rem
-		>div
-			display: flex
-			justify-content: center
+		margin-left: 25%
+		margin-right: 25%
+		>h3
+			font-weight: bold
 			margin-bottom: 1rem
-			>h3
-				font-weight: bold
-				margin-top: 1rem
-				text-transform: uppercase
+			margin-top: 1rem
+			text-align: left
+			text-transform: uppercase
+		>div
+			margin-bottom: 4rem
 			>div
-				flex: 1
-				text-align: left
\ No newline at end of file
+				display: flex
+				justify-content: center
+				margin-bottom: 1rem
+				>div
+					flex: 1
+					text-align: left
+				>div:nth-child(1)
+					flex: 2
+				>div:nth-child(3)
+					text-align: right
```

Comment:
I'm not crazy about this because it messes with the proportions of the content, though I understand why you did it. Don't worry about it for now. We'll sort it out once we decide on the best approach out of margins/flex/grid.